src/stravalib/tests/integration/test_client.py

```diff
@@ -813,6 +814,7 @@ def test_get_activities(
     activity_list = list(client.get_activities(**kwargs))
     assert len(activity_list) == expected_n_activities
     if expected_n_activities > 0:
+        assert isinstance(activity_list[0], SummaryActivity)
```

Comment:
So now, the response from `get_activities()` doesn't have the confusing, always-`None` `laps` attribute anymore.

Reply:
I just tested locally. this works great. i only got information that i could actually retrieve back! 
NOTE: i rarely use laps so at some point i'll create an activity with laps to fully test this. but i don't see an empty laps attr anymore in the response